### PR TITLE
Missings tags

### PR DESCRIPTION
**Pull request by a-saran**

### CODE REVIEW

**Comment by Kameshwaran** [comment]
Looks good to me.

File: gatsby-node.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ const createPages = (type, postArray, createPage) => {
   const groupedPosts = groupPostsByUnique(type, postArray)
   // returns {['tag']: [post1, post2], ['tag2']: [post3, post4]}
   Object.entries(groupedPosts).forEach(([typeValue, postGroup], index) => {
-    typeValue = typeValue.split(" ").join("-")
+    typeValue = typeValue.replace(/\.|\s/g, '-').toLowerCase()
```
Comment (sanandnarayan) [inline gatsby-node.js:37]: so if you do lowercase on the URL, then you need to do the same in preview-template and blog-post-layout template where the tag links are rendered.

**Review by sanandnarayan — CHANGES_REQUESTED** [review]
so if you do lowercase on the URL, then you need to do the same in preview-template and blog-post-layout template where the tag links are rendered.

@asaran97

File: gatsby-node.js (hunk 1 above)
Comment (Kameshwaran) [reply gatsby-node.js:37]: Good point. I should read the code in depth it seems.

**Comment by Kameshwaran** [comment]
@sanandnarayan done with the changes. pls merge when you are free.

**Review by sanandnarayan — APPROVED** [review]

**Pull request merged**